package.json

```diff
@@ -19,50 +19,56 @@
   ],
   "license": "MIT",
   "scripts": {
-    "bundle": "jspm bundle-sfx index.js ./public/js/bundle-sfx.js -m",
-    "coverage": "istanbul cover _mocha -x app/model.js --  --compilers js:babel-core/register --recursive -r jsdom-global/register",
-    "example": "npm run bundle && PORT=3001 nodemon server.js",
-    "start": "npm run bundle && node server.js",
-    "test": "mocha --compilers js:babel-core/register --recursive -r jsdom-global/register"
+    "coverage": "istanbul cover _mocha -x src/model.js --  --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "start": "node example/server.js",
+    "test": "mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "build": "rimraf dist && NODE_ENV=production webpack --progress --profile --colors && NODE_ENV=production",
+    "production": "npm run build && NODE_ENV=production node example/server.js"
```

Comment:
this command runs the example using the production code right? not in dev mode to be precise